tools/torch_quant/setup.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2023 The BladeDISC Authors. All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from setuptools import find_packages, setup
+from torch_quant import version
+
+setup(
+    name='torch_quant',
+    version=version.__version__,
+    author='Alibaba PAI Team',
+    description='PyTorch Quantization Toolkit For BladeDISC',
+    url='https://github.com/alibaba/BladeDISC',
+    packages=find_packages(include=["torch_quant*"]),
```

Comment:
Nit: we might need to exclude the `tools` package from `torch_blade`'s package

Reply:
`pytorch_blade` and `tools` are independent already?

Reply:
OK, I think I misread the directory location before